scripts/markdown-link-check-config.json

```diff
@@ -2,6 +2,9 @@
     "ignorePatterns": [
         {
             "pattern": "^https?://localhost($|[:/].*)"
+        },
+        {
+            "pattern": "^.?[/][\\w].*"
```

Comment:
What regex type is this? Do you know if the word character supports the character set for filenames, or should we make this more broad?

Also I don't think you want to escape `\w`.

Reply:
\w supports alphanumeric characters and '_'. Maybe we should include '.' as well? Open to suggestions for how restrictive we want this to be. 

I think we do need the `\\w`, tough - according to jq, just a single `\w` results in invalid json.

Reply:
Thinking more about the character set that should be good for now. Since the long term plan is adding support in the linter library we can add additional cases here if needed at a later time